apisix/plugins/consumer-restriction.lua

```diff
@@ -67,26 +83,30 @@ function _M.check_schema(conf)
     return true
 end
 
+
 function _M.access(conf, ctx)
-    if not ctx.consumer then
+    if not conf.type then
         return 401, { message = "Missing authentication or identity verification." }
     end
 
+    local type_id = type_funcs[conf.type](ctx)
+    core.log.warn("type_id: ", type_id)
```

Comment:
If this log is only useful to developers, we cannot use the "warning" log level here.